tensorflow/compiler/xla/service/gpu/cusolver_context.cc

```diff
@@ -17,6 +17,22 @@ limitations under the License.
 
 #include "tensorflow/compiler/xla/util.h"
 
+#if !defined(TENSORFLOW_USE_ROCM)
+
+using gpuStream_t = cudaStream_t;
+
+#else
+
+#include "rocm/include/hip/hip_complex.h"
+
+using gpuStream_t = hipStream_t;
+
+#define cusolverDnCreate rocblas_create_handle
```

Comment:
where are these defines used?

Reply:
If it's used below in unchanged code, I think it's preferable to introduce a more platform-agnostic name (`solverSetStream`?) and use that in the same way you define the stream above.

Reply:
> where are these defines used?

just in a couple of routines - 
* https://github.com/tensorflow/tensorflow/pull/50297/commits/632ec990fc9d6bdffd09864a063c8f89a219cd8a#diff-f68a0d118842bbd73c14a1536dc669f953beb2ee0c9c124bc1bc5dad648df6a7R165
* https://github.com/tensorflow/tensorflow/pull/50297/commits/632ec990fc9d6bdffd09864a063c8f89a219cd8a#diff-f68a0d118842bbd73c14a1536dc669f953beb2ee0c9c124bc1bc5dad648df6a7R198

I can create `#if TENSORFLOW_USE_ROCM` versions for them and avoid the #defines altogether (I think).

re the platform agnostic naming..the entire file and its contents are CUDA centric ( `cusolver_context.cc`, `cusolver_rewriter.cc`, etc plus all the CU* names contained within). Renaming the files and all the cu* names within them, is not hard but definitely tedious :) It is likely that the changes will spill to other files too (which call the public interfaces from these files)...so would like to avoid going this route

Reply:
> I can create #if TENSORFLOW_USE_ROCM versions for them and avoid the #defines altogether (I think).

@cheshire pushed out a commit with the above change...let me know if this version is peferable.